web/packages/plugins/apps/analysis-platform/src/modules/Analysis/AnalysisCreate.tsx

```diff
@@ -11,32 +13,52 @@ import { NotificationManager } from 'react-notifications'
 import { TAnalysis } from '../../Types'
 
 export const AnalysisCreate = (): JSX.Element => {
+  const { asset_id } = useParams<{
+    asset_id: string
+  }>()
+  const settings = useContext(ApplicationContext)
   const { tokenData, token } = useContext(AuthContext)
   const user = getUsername(tokenData) || 'NoLogin'
   const [isLoading, setIsLoading] = useState<boolean>(false)
+  const createdAt = new Date().toISOString()
```

Comment:
This might not be the right time. Move it into the "handleCreateAnalysis" function?